dms/models/dms_category.py

```diff
@@ -21,11 +22,9 @@ class Category(models.Model):
     _rec_name = "complete_name"
 
     # ----------------------------------------------------------
-    # Database
+    # Fields
```

Comment:
As this comment format existed previously (which I have already said that seems too many lines for one comment), and you are changing it, I think you can reduce it as recommended or remove it all.